tests/ut/azure_iot_hub_client_properties_ut.c

```diff
@@ -317,12 +344,53 @@ static void testAzureIoTHubClientProperties_GetPropertiesVersion_Success( void *
     assert_int_equal( ulVersion, 5 );
 }
 
+static void testAzureIoTHubClientProperties_GetPropertiesVersionGetDocument_Success( void ** ppvState )
+{
+    AzureIoTHubClient_t xTestIoTHubClient;
+    AzureIoTJSONReader_t xJSONReader;
+    AzureIoTJSONTokenType_t xTokenType;
+    AzureIoTHubMessageType_t xResponseType = eAzureIoTHubPropertiesGetMessage;
+    uint32_t ulVersion;
+
+    assert_int_equal( AzureIoTJSONReader_Init(
+                          &xJSONReader,
+                          ucTestJSONGetPayload,
+                          strlen( ucTestJSONGetPayload ) ),
```

Comment:
can be use sizeof()-1

Reply:
same thing all other places where we are using arrays, where size is known to compiler

Reply:
leaving as is as compiler will very likely convert this into a number